securedrop-admin

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python2
+"""
+SecureDrop Admin Toolkit.
+
+For use by administrators to install, maintain, and manage their SD
+instances.
+"""
+
+import argparse
+import subprocess
+import os
+import sys
+import getpass
+import logging
+
+sdlog = logging.getLogger('SD')
+
+SD_DIR = os.path.dirname(os.path.realpath(__file__))
+ANSIBLE_PATH = os.path.join(SD_DIR, "./install_files/ansible-base")
+SITE_CONFIG = os.path.join(ANSIBLE_PATH, "group_vars/all/site-specific")
+VENV_DIR = os.path.join(SD_DIR, ".venv")
+VENV_ACTIVATION = os.path.join(VENV_DIR, 'bin/activate_this.py')
+
+
+def logger_setup(verbose=False):
+    """ Configure logging handler """
+    # Set default level on parent
+    sdlog.setLevel(logging.DEBUG)
+    level = logging.DEBUG if verbose else logging.INFO
+
+    stdout = logging.StreamHandler(sys.stdout)
```

Comment:
Agree with @msheiny's pushback on this one; while the review comment is valid, it's important to keep the overall context of the project in mind when reviewing code. We're all trying to make progress, and that means keeping issues, PR's, and reviews focused. This is obviously a balancing act, but @fowlslegs I think that a lot of your more recent comments on this PR would've been better addressed by filing follow-up issues (e.g. "improve logging in securedrop-admin", "securedrop-admin dependency upgrade story", etc.). That's especially true of these comments on the logging, where the "correct/best" approach is highly subjective and up for debate.

We're all trying to make SecureDrop's development and release process more "agile" and productive, and a big part of that is trying to find the balance between having reviews that improve code quality while avoiding reviews that hold up progress or frustrate collaborators. This is obviously something we all contribute to, so I'm not trying to call you out unduly @fowlslegs—we all have to work on this going forward, myself included.